texthero/preprocessing.py

```diff
@@ -17,6 +17,44 @@
 
 from typing import List, Callable
 
+# REGEX pattern constants
+DIGITS_BLOCK = r"\b\d+\b"
+PUNCTUATION = rf"([{string.punctuation}])+"
+STOPWORD_TOKENIZER = r"""(?x)                          # Set flag to allow verbose regexps
+                    \w+(?:-\w+)*                              # Words with optional internal hyphens 
+                    | \s*                                     # Any space
+                    | [][!"#$%&'*+,-./:;<=>?@\\^():_`{|}~]    # Any symbol 
+                    """
+ROUND_BRACKETS = r"\([^()]*\)"
+CURLY_BRACKETS = r"\{[^{}]*\}"
+SQUARE_BRACKETS = r"\[[^\[\]]*\]"
+ANGLE_BRACKETS = r"<[^<>]*>"
+HTML_TAG = r"""(?x)                    # Turn on free-spacing
+            <[^>]+>                             # Remove <html> tags
+            | &([a-z0-9]+|\#[0-9]{1,6}|\#x[0-9a-f]{1,6}); # Remove &nbsp;
+            """
+
+URLS = r"http\S+"
+TAGS = r"@[a-zA-Z0-9]+"
+HASHTAGS = r"#[a-zA-Z0-9_]+"
+
+
+def _get_pattern_for_tokenisation(punct: str) -> str:
+    """
+    Return the standart tokenisation pattern
```

Comment:
word `standart` does not mean anything for most of the users ...
we should come up with a better explanation; i.e mentioning both what does this function does and why, which problem it solves

Reply:
I have updated the comment. I hope it will be clearer now 🐰